sdk/containerregistry/azure-containerregistry/dev_requirements.txt

```diff
@@ -3,4 +3,5 @@
 ../../core/azure-core
 ../azure-mgmt-containerregistry
 aiohttp>=3.0; python_version >= '3.5'
-azure-identity
\ No newline at end of file
+azure-identity
+msrestazure>=0.4.11
```

Comment:
Why would you need msrestazure is any way?

Reply:
I need it for tests. The best way I could get to change the management client to target foreign clouds used msrestazure.